### PR TITLE
Upgrade PostgreSQL to 9.4

### PR DESCRIPTION
**Pull request by markbreedlove**
Upgrade PostgreSQL to 9.4, getting the package from the Global Development Group repository.

This includes upgrade notes and a playbook for upgrading an old system.

This changeset is targeted for a specific future release, 2.2.0.  A minor version number is assumed because this is an upgrade of server software and there is no backwards-incompatibility in the usage or maintenance of the `automation` playbooks and variable files themselves. I'm open to other opinions on this matter, however.

This branch needs to be rebased on `develop` after https://github.com/dpla/automation/pull/130 is merged.

### CODE REVIEW

File: README-upgrade-2.2.0.txt [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+9.4 in Release 2.2.0.  This change requires an upgrade of PostgreSQL's tables.
+
+PostgreSQL will copy table and index files when it performs the upgrade, so you
+should make sure that the storage volumes that hold your tablespaces are less
+than half-full.  You can delete the old files when the upgrade is finished,
+as detailed below.
+
+The upgrade will end with an error if your old system databases were created
+with encodings other than UTF8, which has apparently happened within development
+VMs in the `ingestion` inventory (using the Ingestion 2 playbooks).
```
Comment (no-reply) [inline README-upgrade-2.2.0.txt]: Ouch! Thanks for documenting all the upgrade paths for this encoding issue.

**Comment by no-reply** [comment]
This looks good!  :elephant: :nine: :point_right: :four:  :rocket:

**Comment by no-reply** [comment]
I'll let @anarchivist get a review in, still.

**Comment by anarchivist** [comment]
I think this looks good, and I'll test over the weekend (or once i'm off this plane!).

**Comment by anarchivist** [comment]
OK, I'm +1 on this. Thanks! Do we want to :squash:?

**Comment by markbreedlove** [comment]
Yes, I'll squash it, as soon as possible this morning, after dealing with some prerequisites that need to be merged to `develop` first.

**Comment by anarchivist** [comment]
If I'm not mistaken, with dpla/frontend#76, dpla/primary-source-sets#72, and dpla/automation#132 all merged, this should be ready to squash and merge. @markbreedlove I'll hold off and let you confirm in the morning.

**Comment by markbreedlove** [comment]
I just need to modify the upgrade instructions before we merge.  Working on that and doing what I hope is one final test.

**Pull request merged**